pom.xml

```diff
@@ -199,7 +199,6 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>4.12</version>
-            <scope>test</scope>
```

Comment:
Why remove the test scope?

Reply:
I use assert in code.

Reply:
Isn't it better to use Javas own assert https://stackoverflow.com/questions/273485/can-javas-assert-statement-allow-you-to-specify-a-message and not including a test library for this reason?

Reply:
Done.